acceptor.lisp

```diff
@@ -330,6 +333,16 @@ they're using secure connections - see the SSL-ACCEPTOR class."))
   (mp:process-kill (acceptor-process acceptor))
   acceptor)
 
+#-lispworks
+(defun wake-acceptor-for-shutdown (acceptor)
+  "Dials into the accept loop to wake it from select."
```

Comment:
"Dials into" is not quite a clear description of what is happening.  Please reword so that the docstring describes that a dummy connection is created and closed to force the server to check the shutdown request flag.